components/member-profile/contribution-type/contributions-type.module.scss

```
@@ -36,3 +36,9 @@
   margin: 20px 0;
   border-color: lightgray transparent transparent ;
 }
+
+@media (max-width: 575.98px) {
+  .container {
+    margin: 10px;
```

Let's use `rem` here also?
```suggestion
    margin: .625rem;
```

---

As discussed, this is not needed.